libs/Matchmaking/test/Matchmaking.PeerDiscovery.Test/ITransportBuilder.cs

```diff
@@ -19,9 +20,11 @@ internal interface ITransportBuilder
 
     internal class MemoryTransportBuilder : ITransportBuilder
     {
+        private readonly ushort port_ = Utils.NewPortNumber.Calculate();
+
         public IPeerDiscoveryTransport MakeTransport(int userIndex)
         {
-            return new MemoryPeerDiscoveryTransport(userIndex);
+            return new MemoryPeerDiscoveryTransport(port_);
```

Comment:
I wonder should the other backends use the utility & not bother passing userIndex.
Would it be any great loss to drop it, or useful for debugging/logging?

Reply:
Not sure if I am following - the other backends use the same utilities already, userIndex is only used there to generate different IP addresses (here is ignored because unneeded). At the moment userIndex could be replaced by another counter I suppose, but it doesn't harm so I'd rather leave it until the next refactoring.

Reply:
I guess yes, it seems strange to be passing userindex when the transport builders could just as easily use an internal counter. OK to leave as is for now